src/Main.java

```diff
@@ -1,9 +1,250 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Main {
+
+    //Zadatak 1a.
+
+    public static int addNumbers(int numberA, int numberB) {
+        return numberA + numberB;
+
+    }
+
+    public static int subtractNumbers(int numberA, int numberB) {
+        return numberA - numberB;
+    }
+
+    public static int multiplyNumbers(int numberA, int numberB) {
+        return numberA * numberB;
+    }
+
+    public static int divideNumbers(int numberA, int numberB) {
+        return numberA / numberB;
+    }
+
     public static void main(String[] args) {
 
+        System.out.println("Task 1a: ");
+
+        int numberA = 10;
+        int numberB = 5;
+
+        System.out.println("numberA + numberB = " + numberA + " + " + numberB + " = " + addNumbers(numberA, numberB));
+        System.out.println("numberA + numberB = " + numberA + " - " + numberB + " = " + subtractNumbers(numberA, numberB));
+        System.out.println("numberA + numberB = " + numberA + " * " + numberB + " = " + multiplyNumbers(numberA, numberB));
+        System.out.println("numberA + numberB = " + numberA + " / " + numberB + " = " + divideNumbers(numberA, numberB));
+
+
+        //Zadatak 1aa.
+
+        System.out.println("\nTask 1aa: ");
+
+        Scanner scanner = new Scanner(System.in);
```

Comment:
declaration of objects to be used through out the project go at the begining of class